examples/abbas_work_area/palindrome.py

```diff
@@ -0,0 +1,15 @@
+user = input("Enter: ").strip().replace(" ", "").lower()
+punc = []
```

Comment:
Please consider using a meaningful variable name like list_of_punctuation_characters or something like this